snappass/main.py

```diff
@@ -11,7 +11,7 @@
 from urllib.parse import unquote_plus
 from urllib.parse import urljoin
 from distutils.util import strtobool
-from flask_babel import Babel
+from flask_babel import Babel, _
```

Comment:
why importing `_`?

Reply:
this caused linter error

Reply:
I didn't manage to get the Jinja templates translated without importing `_`. Came accross the solution [here](https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-xiii-i18n-and-l10n).

Should i [in-line ignore](https://flake8.pycqa.org/en/stable/user/violations.html#in-line-ignoring-errors) this linter error? Other suggestions?

Reply:
Got it. In-line ignoring is fine, please add explanation why doing that.